### DISCUSSION TITLE
Anyone knows how to link my store items into world?

### DISCUSSION THREAD

**Discussion by rebeccakzpt** [naverz/zepeto-studio-kor#715] [5. Q&A]
The official tutorial is very brief...and the international forum is dead :(
Please lend me a hand if you can. Thank you so much!!

**Comment by Zepeto-TechEvangelist** [comment]
Hello, please refer to the guide below.
https://studio.zepeto.me/kr/guides/in-world-purchases
Thank you

**Reply by rebeccakzpt** [reply]
Thanks, but that doesn't include linking my creator items :(

**Reply by v2form** [reply]
> Thanks, but that doesn't include linking my creator items :(

Im also stuck in that part now

**Reply by Zepeto-TechEvangelist** [reply]
The function to change the costume of ZEPETO characters is not yet available in the official API.
Please understand that it will be provided in the future and the application period is undecided. 

Thank you.
